src/utils/body.test.ts

```diff
@@ -1,4 +1,9 @@
 import { parseBody } from './body'
+import type { BodyData } from './body'
```

Comment:
This is cool PR!

I think you can just merge the imports in one import like this

```ts
import { parseBody, type BodyData } from './body'
```

I think we need to setup eslint to avoid imports to the same file @usualoma  @yusukebe

Reply:
Thank you! That's certainly better. 
I made the change in eecfc43.

Reply:
@fzn0x 

> I think we need to setup eslint to avoid imports to the same file @usualoma  @yusukebe 

That's great! I'll work on it soon.